projects/dsp-ui/src/lib/viewer/representation/upload-form/upload-form.component.ts

```diff
@@ -0,0 +1,134 @@
+import { Component, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormControl, FormGroup, Validators } from '@angular/forms';
+import { NotificationService } from '../../../action/services/notification.service';
+import { UploadedFileResponse, UploadFileService } from '../../services/upload-file.service';
+
+@Component({
+    selector: 'dsp-upload-form',
+    templateUrl: './upload-form.component.html',
+    styleUrls: ['./upload-form.component.scss']
+})
+export class UploadFormComponent implements OnInit {
+
+    @Input() readonly resourceTyoe = 'Image'; // only StillImageRepresentation supported so far
```

Comment:
typo: `resourceTyoe`

Why is this readonly? Since it is an `@Input`, it could get a value from the parent.

Reply:
Thanks for that. It was readonly before it became an `@Input`. Again I'm not yet sure how this gonna work. I understand this is how the component could recognise type of resource and base on that restrict upload to the list of filetypes. I will remove it.

Reply:
Maybe there is no need for an `@Input` as long as there is just one supported type. However, it is good that you already made a design that would support other types as well.

Reply:
So, @kilchenmann said we need that Input, then bringing it back with b4143c5.

Reply:
Ok, fine with me (without `readonly`)